### DISCUSSION TITLE
Still active?

### DISCUSSION THREAD

**Discussion by Hutch79** [JordanKnott/taskcafe#175] [General]
Hey, 
I just stumbled over this project and think it has grate potential.
I searched a long time for something to manage my to-do's in a Trello like way, and TaskCafe is one of the best looking I found.

Unfortunately, I noticed a few issues with the docker version and Firefox, like clicking an element in a list not opening the edit window and other button/links not loading.
For this reason, I took a stop here and wonder now if the development is still active.

The main reason for this question is the activity of the Issues as well as the commits.
Last activity in the code was on the second of Septembers, 2022 which is over half a year ago.

If this repo really is inactive, does anyone know a fork which is in active development?

**Comment by ServerGeek82** [comment]
https://github.com/JordanKnott/taskcafe/discussions/162

Author has not forgotten about the project.

**Reply by Hutch79** [reply]
Oh, thanks for the link!
Haven't seen this one.

I'm now using an alternative, but TaskCafe still is the preferred option ^^

**Reply by ServerGeek82** [reply]
I'm using planka for now. But I want to see where this one goes.

**Reply by Hutch79** [reply]
Same, I also use Planka.
I only miss the Public boards ^^

**Reply by foxhoundv** [reply]
I just checked out that other one... I hope this one gains more traction. I don't like Planka not being able to mark something as done in it's own card area, though I do like adding separate viewable boards in the same project as a whole.

**Reply by ServerGeek82** [reply]
After trying kanban planka was a good alternative. I have a compose script waiting made for this in my CI CD pipeline when an update happens.

**Comment by JordanKnott** [comment]
I am currently in the middle of rewriting the entire project, though only whenever I have both the time and desire.

I have thought about dedicating some time to closing out some of the currently open issues but I'm not sure if its worth it in the long run.

**Reply by foxhoundv** [reply]
@JordanKnott Wow! May I ask why the complete rewrite? I do enjoy what you have though I completely understand if there is certain limitations based on the code language used...

**Reply by JordanKnott** [reply]
I'm still happy with the languages I've chosen for the project, just less so with the overall architecture. I started this project to learn more about React and it shows - my hope for the rebuild is that it will be much more stable. 

The UI and everything will remain the same (though more consistent) - the only thing changing in the rewrite is how the frontend code is structured.

**Reply by foxhoundv** [reply]
Whew, I wish I was a programmer to be able to help... I am the type who can read code and for the most part understand what it is doing, but when it comes to creating code and making the connections... not my forte.

**Reply by thomastthai** [reply]
Go + HTMX may be worth checking out for the rewrite.
